README.md

```diff
@@ -1,142 +1,88 @@
-  provided by 3rd parties.
-
-  * If you need `ssh` to work behind a proxy, you also need [`proxytunnel`](http://proxytunnel.sourceforge.net/) installed (available as `proxytunnel` package for Ubuntu, for example).
-
-* The `balena sync` command currently requires `rsync` (>= 2.6.9) to be installed:
-  * Linux: `apt-get install rsync`
-  * MacOS: [Xcode command-line tools](https://developer.apple.com/xcode/features/) or [homebrew](https://brew.sh/)
-  * Windows: use MinGW as described in the Windows Support section.
-
-### Windows Support
-
-We aim at supporting the standard Windows Command Prompt (`cmd.exe`) and the Windows [PowerShell](https://docs.microsoft.com/en-us/powershell/scripting/getting-started/getting-started-with-windows-powershell?view=powershell-6).
-
-Some CLI commands like `balena sync` and `balena ssh` have not been thoroughly tested with the standard Windows shells. We are aware of users having a good experience with alternative shells, including:
-
-* Microsoft's [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about) (a.k.a. Microsoft's "bash for Windows 10").
+* Microsoft's [Windows Subsystem for Linux](https://docs.microsoft.com/en-us/windows/wsl/about)
+  (a.k.a. Microsoft's "bash for Windows 10").
 * [Git for Windows](https://git-for-windows.github.io/).
-* [MinGW](http://www.mingw.org)
-  1. Install [MinGW](http://www.mingw.org).
-  2. Install the `msys-rsync` and `msys-openssh` packages.
-  3. Add MinGW to the `%PATH%` if this hasn't been done by the installer already. The location where the binaries are places is usually `C:\MinGW\msys\1.0\bin`, but it can vary if you selected a different location in the installer.
-  4. Copy your SSH keys to `%homedrive%%homepath\.ssh`.
-  5. If you need `ssh` to work behind the proxy you also need to install [proxytunnel](http://proxytunnel.sourceforge.net/)
+* [MinGW](http://www.mingw.org): install the `msys-rsync` and `msys-openssh` packages too.
 
-### Login
+### Logging in
+
+Several CLI commands require access to your balenaCloud account, for example in order to push
+a new release to your app. Those commands require creating a CLI login session by running:
 
 ```sh
 $ balena login
 ```
 
-_(Typically useful, but not strictly required for all commands)_
-
-### Run commands
-
-Take a look at the full command documentation at [https://balena.io/docs/tools/cli/](https://balena.io/docs/tools/cli/#table-of-contents
-), or by running `balena help`.
-
-### Bash completions
-
-Optionally you can enable tab completions for the bash shell, enabling the shell to provide additional context and automatically complete arguments to`balena`. To enable bash completions, copy the `balena-completion.bash` file to the default bash completions directory (usually `/etc/bash_completion.d/`) or append it to the end of `~/.bash_completion`.
-
-FAQ
----
-
-### Where is my configuration file?
```

Comment:
The FAQ section was moved from README.md to the pre-existing TROUBLESHOOTING.md, with a link to it under the section "Support, FAQ and troubleshooting".